src/App.js

```diff
@@ -1,20 +1,28 @@
 import React from 'react';
-import {BrowserRouter as Router, Route, Switch} from 'react-router-dom';
+import { HashRouter as Router, Route, Switch} from 'react-router-dom';
```

Comment:
I was facing issues of reloading the window and then the page throws a blank page. When the URL is copied and pasted in another tab it would also show a blank page. Therefore this made clicking the password  reset link render a blank page which is practically unacceptable. Using HashRouter sorts the issue. Here is the [resource](https://stackoverflow.com/questions/27928372/react-router-urls-dont-work-when-refreshing-or-writting-manually) I used to find a solution